trade-finance-manager-ui/server/controllers/utilisation-reports/find-reports-by-year/index.ts

```diff
@@ -70,7 +71,18 @@ export const getFindReportsByYear = async (req: Request, res: Response) => {
       });
     }
 
-    return renderPreviousBankReportsByYear(res, { user });
+    const { bankName, year, reports } = await api.getReportsByBankAndYear(userToken, bankIdQuery!, yearQuery!);
```

Comment:
I'm not sure about using the non-null assertion operator here, thoughts on alternatives?

Reply:
Ahh this is interesting... Perhaps we could have the `validateSearchInput` function return the values once validated as `string`s and use those? That way if ever the validation changes to not check it will cause a type error on this line?